pkg/plugin/v3/scaffolds/internal/templates/main.go

```diff
@@ -260,6 +264,9 @@ func main() {
 
 	%s
 
+	_ = mgr.AddHealthzCheck("check", func(_ *http.Request) error { return nil })
+	_ = mgr.AddReadyzCheck("check", func(_ *http.Request) error { return nil })
```

Comment:
Required to check we really need to add it

Reply:
I have verified that we need this change to actually serve the readyz and healthz endpoints